lib/honeybadger/config/defaults.rb

```diff
@@ -343,6 +343,11 @@ class Boolean; end
         description: 'Enable/Disable automatic breadcrumbs from log messages.',
         default: true,
         type: Boolean
+      },
+      :'insights.enabled' => {
```

Comment:
Something to keep in mind for the extra optional plugin config—if it's not explicitly defined in the defaults, people won't be able to configure the plugin values via environment variables, since [the env config uses the defaults to know what to look for](https://github.com/honeybadger-io/honeybadger-ruby/blob/ea0101da818354d651bb7a52a97c8a38004d71b9/lib/honeybadger/config/env.rb#L7).

Reply:
Oh wow, thank you. This was in the back of the mind, to figure out how things are configured through ENV's.

Reply:
I'll add some defaults for the plugin specific configurations.